README.md

```diff
@@ -1,25 +1,45 @@
-# Norwegian Earth System Model (NorESM)
-
-
-- [NorESM-1 documentation](https://noresm-docs.readthedocs.io/en/noresm1)
-- [NorESM-2.0 documentation](https://noresm-docs.readthedocs.io/en/noresm2/)
-- [Latest NorESM documentation](https://noresm-docs.readthedocs.io/en/latest) currently showing mainly NorESM1 documentation
-
-# Branches in NorESM
-
-This repository has the following NorESM branches:
-
-* master - combines and contains recent developments, based on the latest release (at this stage NorESM2.0.2)
-
-* master_cesm - Community Earth System Model master fork
-
-* noresm1 - contains a copy of old wiki NorESM1 documentation, refering to https://github.com/metno/noresm NorESM1 code, and incompletely to the development code under https://github.com/metno/noresm-dev (restricted access), used in 2019 for CMIP6 simulations
-
-* noresm2 - contains NorESM2.0.X documentation, code and configurations as used for most NorESM2-LM and NorESM2-MM CMIP6 simulations, with tags:
-
-  - release-noresm2.0.4 - Release of NorESM2.0.4 9th of April 2020, available to reproduce CMIP6 results of NorESM2. Additional Fram and Betzy settings included.
-  - release-noresm2.0.3 - Release of NorESM2.0.3 7th of April 2021, available to reproduce CMIP6 results of NorESM2. Betzy settings included 
-  - release-noresm2.0.2 - Release of NorESM2.0.2 20th of July 2020, available to reproduce CMIP6 results of NorESM2
-  - release-noresm2.0.1 - NorESM2 version, initial preliminary release of CMIP6 version, April  2020 
+![NorESM logo](doc/img/NORESM-logo.png)
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3905091.svg)](https://doi.org/10.5281/zenodo.3905091)
+
+[![Documentation Status](https://readthedocs.org/projects/noresm-docs/badge/?version=noresm2)](https://noresm-docs.readthedocs.io/en/noresm2/?badge=latest)
+
+# The Norwegian Earth System Model version 2 (NorESM2)
+The **noresm2** branch contains the NorESM2.0.X documentation, code and configurations as used for most  <br />
+NorESM2-LM and NorESM2-MM CMIP6 simulations
+
+The NorESM-2.0 documentation: https://noresm-docs.readthedocs.io/en/noresm2/
+
+## Releases in noresm2
+- release-noresm2.0.5
+    - Release of NorESM2.0.5 Xth of April 2021
+    - Available to reproduce CMIP6 results of NorESM2
+    - Updated NorESM2.0 documentation.
+    - https://github.com/NorESMhub/NorESM/releases/tag/release-noresm2.0.5
+
+- release-noresm2.0.4
```

Comment:
I was thinking to the text "release-noresm2.0.4" into a link
[release-noresm2.0.4](https://github.com/NorESMhub/NorESM/releases/tag/release-noresm2.0.4)